### DISCUSSION TITLE
Anti AFK bypass permission for vanilla idle kick

### DISCUSSION THREAD

**Discussion by TechnicallyCoded** [PurpurMC/Purpur#656] [__Completed__] [answered]
**The feature request**
At the moment there is no way to bypass the anti afk kick of vanilla minecraft. Any player with the suggested permission would not be kicked from the server for idling. (something like purpur.preventidletimeout)

**Suggesting this to purpur and not paper**
Suggesting this to purpur since it's more of a feature rather than an optimization and seems to fall in line with the idea of adding permissions to change vanilla behaviors (such as purpur.joinfullserver)

**Comment by katekerllenevich** [comment, marked as answer]
Added in 161dcf0

**Reply by TechnicallyCoded** [reply]
Awesome! Thank you :)
